srcCpp/perftest_publisher.cxx

```diff
@@ -135,6 +135,30 @@ int perftest_cpp::Run(int argc, char *argv[])
     }
 }
 
+const DDS_ProductVersion_t perftest_cpp::GetDDSVersion()
+{
+    return NDDSConfigVersion::get_instance().get_product_version();
+}
+
+const Perftest_ProductVersion_t perftest_cpp::GetPerftestVersion()
+{
+    return _version;
+}
+
+void perftest_cpp::PrintVersion()
+{
+    Perftest_ProductVersion_t perftestV = perftest_cpp::GetPerftestVersion();
+    DDS_ProductVersion_t ddsV = perftest_cpp::GetDDSVersion();
+
+    printf("RTI Perftest: %d.%d.%d (RTI Connext DDS %d.%d.%d)\n",
+           perftestV.major,
+           perftestV.minor,
+           perftestV.release,
+           ddsV.major,
+           ddsV.minor,
+           ddsV.release);
+}
+
```

Comment:
Could you implement it in the rest of languages?

Reply:
Done.